### PR TITLE
update: add helper_method in filter select

### PR DESCRIPTION
**Pull request by Mahaveer1141**
## What does this change do?
- add `helper_method` in filter select

## Why is this change needed?
- there's no `helper_method` in filter select.

## How were the changes done?
- add `helper_method` field in filter.
- for `single_select` and `multi_select`, check if `helper_method` is present then get options from it otherwise get options from collection.

### CODE REVIEW

File: lib/cm_admin/view_helpers/filter_helper.rb [hunk 1]
```diff
@@ -28,26 +27,26 @@ def add_filters_dropdown(filters)
...
+                  concat(content_tag(:div, class: 'pointer list-item', data: { behaviour: 'filter-option', filter_type: "#{filter.filter_type}", db_column: "#{filter.db_column_name}" }) do
                     tag.span filter.placeholder.to_s.titleize
                   end)
                 end
               end)
             end)
           end)
         end)
-        return
+        nil
```
Comment (anbublacky) [inline lib/cm_admin/view_helpers/filter_helper.rb:42]: @Mahaveer1141 Why id we have empty return and having nil now for methods?

**Review by anbublacky — COMMENTED** [review]
Rest are Ok. You can merge it after the comment is resolved.

File: lib/cm_admin/view_helpers/filter_helper.rb (hunk 1 above)
Comment (Mahaveer1141) [reply lib/cm_admin/view_helpers/filter_helper.rb:42]: @anbublacky It's a rubocop change, we can remove empty returns that will also return nil.

Comment (Mahaveer1141) [reply lib/cm_admin/view_helpers/filter_helper.rb:42]: @anbublacky If we don't return nil, filter UI seems to breaking

Comment (anbublacky) [reply lib/cm_admin/view_helpers/filter_helper.rb:42]: Ok bro

**Review by anbublacky — APPROVED** [review]

**Pull request merged**